common/shlibs

```diff
@@ -4087,11 +4087,11 @@ libumfpack.so.6 SuiteSparse-6.0.1_1
 libecl.so.21.2 ecl-21.2.1_1
 libecm.so.1 ecm-7.0.4_3
 libcliquer.so.1 cliquer-1.22_1
-libSingular-4.3.1.so singular-4.3.1p2_1
-libfactory-4.3.1.so singular-4.3.1p2_1
-libpolys-4.3.1.so singular-4.3.1p2_1
-libomalloc-0.9.6.so singular-4.3.1p2_1
-libsingular_resources-4.3.1.so singular-4.3.1p2_1
+libomalloc-0.9.6.so singular-4.2.1_1
```

Comment:
typo?

Reply:
The soname for `libomalloc` has been the same since `4.2.1_1`, while the others seem to change every version. In practice, it's probably irrelevant since there won't be any program that links only to `libomalloc` (only `sagemath` depends on `singular`). It's annoying that they change the soname every release... I wonder if it's really necessary.

Reply:
I see, sorry for the confusion